src/app/component/organisms/abstract-session-management/abstract-session-management.component.spec.ts

```diff
@@ -0,0 +1,31 @@
+import {ComponentFixture, TestBed} from '@angular/core/testing';
+
+import { AbstractSessionManagementComponent } from './abstract-session-management.component';
```

Comment:
there are spaces in this bracket but not the ones below

Reply:
Wird mit dem Ticket LFD-38 upgedatet.